src/discordcr/mappings/rest.cr

```diff
@@ -2,6 +2,13 @@ require "./converters"
 
 module Discord
   module REST
+    # Enum for controlling emission of JSON key/value pairs where
+    # JSON `null` is a significant value.
```

Comment:
I could probably be more explicit here about how the enum is intended to be used.

"`MaybeField::None` will emit a key and a JSON `null` value. `MaybeField::Unchanged` will not emit a key or a value."